package.json

```diff
@@ -392,6 +392,21 @@
                     "default": false,
                     "description": "Set to true to show timestamps by default in the log viewer."
                 },
+                "vscode-kubernetes.log-viewer.since": {
+                    "type": "integer",
+                    "default": -1,
+                    "description": "The default value for the since field in the log viewer. Set to -1 for all logs."
```

Comment:
Documentation says 0?

Reply:
I can change this to 0 if you'd prefer.

I used `-1` because `0` and `-1` do the same thing in this extension, and the default value of the setting was `-1`. ([Reference](https://github.com/vscode-kubernetes-tools/vscode-kubernetes-tools/pull/1074/files#diff-79cdfcae916cbf064210c646791439ef36aabbbb50c25e5bcd7164f4b4bc810bR343))

The default value was changed in a previous PR I made here:
> In the log viewer, changes the default option for since-input to -1 and since-select to s. (This stills results in getSinceDuration() returning 0 by default because of sinceInput <= 0.)
> * https://github.com/vscode-kubernetes-tools/vscode-kubernetes-tools/pull/1063

I thought `-1` would be more generally understood as unspecified unlike `0` which could be mistaken for 0 seconds for those less familiar with Kubernetes.

Perfectly happy to use the number 0 instead, but probably best to also change the default value back if so.

Reply:
I don't mind whether it's 0 or -1 it was just that the changes to README.md said the default was 0 and I found it confusing that it said different things in the two places.

Reply:
Ahh, my bad! Yeah I should probably update the README too then! ^-^'